R/ZZZ.R

```diff
@@ -16,3 +16,17 @@ if (getRversion() >= "2.15.1")
     "name",
     "section"
   ))
+
+# Install the required font
+.onLoad <- function(libname, pkgname) {
+  if (Sys.info()[1] == "Linux") {
```

Comment:
see comment in R/theme_nhs.R - this could be replaced with a block like (pseudocode)
`if frutiger font exists, then set option: nhsrtheme.font = "Frutiger"`
we could also issue a `warning()` to indicate font is missing (link to a vignette explaining how to install?)

[I seem to remember seeing NHS Digital will provide the frutiger font if you are NHS on request, but that will not be sufficient for us to redistribute]

Reply:
Hi Tom cheers for the comments - was trying to be clever and install it for everyone who used the package…

I’ll remove the `fonts/Frutiger.ttf` file and installation functions in `R/ZZZ.r`.

Then apply some logic in `R/theme_nhs.R` when selecting the font:

Frutiger > Arial > default

Will let you know when it’s sorted for another review.

Reply:
As a work around for now I have just removed the dependency - can think of a better way to incorporate it in future